src/app/shared/components/home/home.component.html

```diff
@@ -0,0 +1,20 @@
+<h1 class="text-uppercase text-center text-bold text-primary">film database</h1>
+
+<div [hidden]="!(searchResult$ | async)">
+    <app-get-movies [checkLoggedIn]="checkLoggedIn" [movie]="searchResult$ | async "></app-get-movies>
+</div>
+
+<!-- <ng-template #moviesSideBar> -->
+<div [hidden]="(searchResult$ | async)" *ngFor="let movie$ of movies ">
```

Comment:
you should subscribe to (searchResult$ | async) once. Try with ngIf at the top

Reply:
Yes this is stupid, is it better to make the subscription in the search.component.ts, because if I use ng if with a sync pipe I will subscribe twice, once when I checked if it's empty and once when I make the loop.

Reply:
Done I've found the ng if + as